components/EntryList.jsx

```diff
@@ -0,0 +1,31 @@
+import EntryCard from "./EntryCard";
+import styled from "styled-components";
+
+export default function EntryList({ entries }) {
+  return (
+    <>
+      <ListContainer>
+        {entries.map((entry) => {
+          return (
+            <EntryCard
+              key={entry._id}
+              type={entry.emotion}
+              intensity={entry.intensity}
+              dateTime={entry.dateTime}
+/*               id={entry._id} */
+            />
+          );
+        })}
```

Comment:
```suggestion
        {entries.map((entry) => (
            <EntryCard
              key={entry._id}
              type={entry.emotion}
              intensity={entry.intensity}
              dateTime={entry.dateTime}
            />
          ))}
```

Use explicit returns only when you need multiple statements, or some local variables, or more complex logic inside the map. Let s use implicit returns when the JSX is simple like here. Also consistency matters. Having two different styles in the same project can subtly confuse people reading the code (eg. your map in `EntryCardWrapper` vs this one.)